odoo/cli/help.py

```diff
@@ -21,6 +24,7 @@ class Help(Command):
 
     def run(self, args):
         load_internal_commands()
+        odoo.modules.initialize_sys_path()
```

Comment:
This should be called inside `load_addons_command`, shouldn't it?

Reply:
Yes, it makes sense.